liquibase-standard/src/main/java/liquibase/parser/core/formattedsql/FormattedSqlChangeLogParser.java

```diff
@@ -142,4 +155,60 @@ private SqlPrecondition parseSqlCheckCondition(String body) throws ChangeLogPars
         }
         throw new ChangeLogParseException("Could not parse a SqlCheck precondition from '" + body + "'.");
     }
+
+    private TableExistsPrecondition parseTableExistsCondition(String body) throws ChangeLogParseException {
+        Matcher tableMatcher = TABLE_NAME_STATEMENT_PATTERN.matcher(body);
+        Matcher schemaMatcher = SCHEMA_NAME_STATEMENT_PATTERN.matcher(body);
+        TableExistsPrecondition tableExistsPrecondition = new TableExistsPrecondition();
+
+        if (tableMatcher.matches()) {
+            if (tableMatcher.groupCount() > 1) {
+                throw new ChangeLogParseException("Multiple table names were specified in tableExists precondition '" + body + "'.");
+            }
+
+            tableExistsPrecondition.setTableName(tableMatcher.group(1));
+            System.out.println(tableMatcher.group(1));
```

Comment:
There are a few of these messages in the `parseTableExistsCondition()` and in `parseViewExistsCondition()`. I imagine these were added for testing purposes when you developed these methods, if that's the case can we please remove them? But if you add them trying to log status or any specific action, please use a logger and add a message of what you want to track.

Reply:
I will remove them. They only were added for some testing purposes.